lib/beaker/host/cisco.rb

```diff
@@ -21,7 +21,7 @@ def skip_set_env?
     #
     # @return nil
     def scp_post_operations(scp_file_actual, scp_file_target)
-      if self[:platform] =~ /cisco-5/
+      if self[:platform] =~ /cisco-nexus/
```

Comment:
Not sure what's gained specifically by gating this for Nexus. Why not instead `if scp_file_actual != scp_file_target`?